scripts/nb2doc/check_nb_docs.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
```

Comment:
As discussed, I'd rather run the existing script then check if any differences have been introduced using `git`, for example running `git diff --quiet doc/modules/ROOT/pages/tutorials/` and checking its exit code.